test/libsolidity/smtCheckerTests/modifiers/modifier_parameters.sol

```diff
@@ -0,0 +1,18 @@
+pragma experimental SMTChecker;
+
+contract C
+{
+	uint s;
+	modifier m(uint a) {
+		// Condition is always true for m(2).
+		require(a > 0);
+		_;
+	}
+
+	function f(uint x) m(x) m(2) m(s) public view {
+		assert(x > 0);
+		assert(s > 0);
+	}
+}
+// ----
+// Warning: (127-132): Condition is always true.
```

Comment:
Does the rest of the warning make it clear that this comes from a call through f?

Reply:
It doesn't... maybe we should add the call path as secondary location.

Reply:
Added in #6310